### ISSUE TITLE
[Feedback] Interactables feature branch feedback

### ISSUE THREAD

**Issue by SimonDarksideJ** [microsoft/MixedRealityToolkit-Unity#2938]
# Overview
Summary of feedback on the Interactables branch:

- [x] Demo scene needs an instruction banner, like the other demo scenes
- [ ] UI elements should align on the screen, consider using an object collection
- [ ] There should be no warnings produced by Unity in the base project.
- [x] Body / Head objects need updating to the new standard.  MixedRealityPlaySpace / Main Camera
- [x] Theme "states" are free text fields, should these not be drop downs?
- [x] (question) What is the difference between "themes" and Input Actions.  Likely a docs thing but needs clearing up.
- [ ] Inspectors should have some "help text" to describe what the "thing" does.  (possibly collapsible as we've seen in other "demos")
> (P.S. Really like the look and feel of the inspectors btw, nice and clean)
- [x] Right-click Create option for Themes and States needs to live under the MixedRealityToolkit option.  E.G. MixedRealityToolkit -> Interacables -> ...
- [x] I got a Unity exception with some of the Interactable inspectors.  Will try and test more to find the specific one
- [x] Consider renaming elements specific to Interactables to be prefixed with Interactable.  E.G. onClickReceiver -> InteractableOnClickReceiver.
Ideally, there should not be a script that you don't know what it is, where it's from and which system it belongs to. Also ProfileItem -> InteractableProfileItem for example.
- [ ] TODO's should be complete or in another Task.  TODO's should not exist in the staging or dev branch, only complete code.
- [ ] We changed the InputSystem lookup to simply MixedRealityManager (soon to be toolkit) .Instance.InputSystem, to make accessing major system easier.  Check current dev branch.

Will try and more tomorrow when I get more time.

**Comment by killerantz** [comment]
Thanks for the great feedback @SimonDarksideJ 

Some of the items listed have been fixed in my latest PR, the others are discussed below.

- The Theme states that are text fields is for the AnimatorTheme. Those states represent the triggers in the Animator. A user could use their own Animator, so these should be easily changable. This is the pattern used by Unity's GUI Button as well.

- The difference between themes and input actions is that themes link object properties to interaction states. Now States are very similar to input actions, but we may not want the granularity of the input states to be surfaced at the input actions level. A way to look at it is this system is setup to provide feedback for how an input action or input in general is occurring.

- Inspectors should have help or introduction text, I'll work on that. 

- I fixed one inspector error in my latest PR. There could be more :)


I'll continue to work on these tomorrow.
